src/custom/dep.cpp

```diff
@@ -330,19 +330,20 @@ static const struct {
     { kModeMyth, "/myth-modules/res/Molphar.svg", {}, -1 },
     // CC0-1.0
     { kModeNonlinearcircuits, "/nonlinearcircuits/res/BOOLs2.svg", {}, -1 },
```

Comment:
It seems that `BOOLs2.svg` has been deleted?

Reply:
Fixed.